docs/ui/schema-conv/spanner-draft.md

```diff
@@ -28,7 +28,7 @@ Users can view detailed information for a table by selecting it from the **Spann
 
 Column tab provides information on the columns that are a part of the selected table. It also provides the option to edit the column wherein a user can modify a column name, delete a column, change the data type of the column, add auto-generation to the column or modify the null property of the column. Once the user is done with required modifications, they can click on **SAVE & CONVERT **and the update would reflect in the session file and across all the components in the database.
```

Comment:
This section needs to be updated with latest screenshots apart from the one updated and details around modifying/adding default values: https://googlecloudplatform.github.io/spanner-migration-tool/ui/schema-conv/spanner-draft.html#column